rs/types/types/src/messages/inter_canister.rs

```diff
@@ -653,6 +655,57 @@ impl From<Response> for RequestOrResponse {
     }
 }
 
+/// A message that acts as a blocker for the message behind it in the `StreamIndexedQueue` of a
+/// `StreamSlice` until the `begin` in a different `StreamSlice` coming from `subnet_id` is verified
+/// to be >= `index`.
+#[derive(Clone, Debug, Eq, PartialEq, Hash, Deserialize, Serialize)]
+#[cfg_attr(test, derive(ExhaustiveSet))]
+pub struct StreamBlocker {
+    pub subnet_id: SubnetId,
+    pub index: StreamIndex,
+}
+
+/// `RequestOrResponse` extended by `StreamBlocker` which exists only in stream slices and streams.
+#[derive(Clone, Debug, Eq, PartialEq, Hash, Deserialize, Serialize)]
+#[cfg_attr(test, derive(ExhaustiveSet))]
+pub enum RequestOrResponseOrBlocker {
+    Request(Arc<Request>),
+    Response(Arc<Response>),
+    Blocker(Arc<StreamBlocker>),
```

Comment:
Just wondering (no strong opinion either way) whether the variant name shouldn't match the type it wraps (whether for consistency or best practices or whatever).